src/components/header/style.module.css

```diff
@@ -470,7 +470,6 @@
 	height: 56px;
 	min-width: 80px;
 	overflow: visible;
-	background: var(--color-brand);
```

Comment:
Sorry, drive-by fix: the height of the search bar is ever so slightly larger than the other header items, and, on mobile, it overlaps (and I mean by 1-2px) into the Ukraine banner above it. We don't need a background here though, it's already sitting on a brand-colored header, so removing this fixes that.